### DISCUSSION TITLE
Error Code?

### DISCUSSION THREAD

**Discussion by MatchstickWorks** [gcormier/megadesk#143] [General]
One day, when I tried to use my configured positions. It would do the number of beeps as before, but then do 4 tones in descending pitch and refuse to move. It goes up and down manually fine, but the memory settings seem to be busted. I've tried turning off and on again. Just curious if there was a list of "error codes" for this based on the tone pattern/pitch.

**Comment by gcormier** [comment]
4 tones indicates there is no stored position at that memory location. Can you try saving again?

**Reply by MatchstickWorks** [reply]
Ah, that was it! Thanks!

**Comment by MatchstickWorks** [comment]
Memory must've been wiped on a power outage. 4 tones = no stored information. Re-saving the position fixed the problem. Thanks gcormier!
